public/app/plugins/datasource/loki/components/types.ts

```diff
@@ -4,3 +4,10 @@ import { LokiDatasource } from '../datasource';
 import { LokiOptions, LokiQuery } from '../types';
 
 export type LokiQueryEditorProps = QueryEditorProps<LokiDatasource, LokiQuery, LokiOptions>;
+
+export interface QueryStats {
```

Comment:
I don't think this is a correct place for `QueryStats` interface as it is not related to loki components, but rather to what loki data source returns. I'd consider moving it to https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/loki/types.ts.